### DISCUSSION TITLE
Arbitrage trading settings with real money.

### DISCUSSION THREAD

**Discussion by BlueCute** [Drakkar-Software/OctoBot#2035] [Q&A]
Hello,

I want to ask how to set up the arbitrage profile to work with real money.
Also want to ask, do I have to have money in two exchanges or is it enough in just one of the exchanges?
would like to try binance and ftx.
Also tried DailyTrading account but it doesn't do anything. It just doesn't trade even though I have pinned 4 exchanges.
I'm asking about the Windows version.
If someone can land a hand will be greatfull.

P.s.
The Linux version does not install.
ERROR: ccxt 1.79.91 has requirement setuptools>=60.9.0, but you'll have setuptools 41.6.0 which is incompatible.
ERROR: Could not install packages due to an EnvironmentError: [Errno 30] Read-only file system: '/snap/python38/22/usr/local'

WARNING: You are using pip version 19.3.1; however, version 22.2.2 is available.

The thing is that i already upgrade everything. And pip version indeed is 22.2 but still getting errors.

Kind Regards,

**Comment by Herklos** [comment]
Hi @BlueCute. Concerning your linux issue ([Errno 30] Read-only file system: '/snap/python38/22/usr/local'), you shouldn't use the python snap install but the `apt` python installation method. It should fix your linux issue.

**Reply by BlueCute** [reply]
Thank you ! I will try this now.

**Reply by BlueCute** [reply]
note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.

That i get when i try python install.

**Comment by BlueCute** [comment]
Has anyone tested the arbitrage profile working or not ?
Or at least someone give advice on how to set up Daily Trading.
Please.
